hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/test/java/org/apache/hadoop/yarn/client/cli/TestRMAdminCLI.java

```diff
@@ -1099,4 +1099,27 @@ public void testNoUnsupportedHACommandsInHelp() throws Exception {
     assertFalse(errOut.contains("-failover"));
     dataErr.reset();
   }
+
+  @Test
+  public void testParseSubClusterId() throws Exception {
+    rmAdminCLI.getConf().setBoolean(YarnConfiguration.FEDERATION_ENABLED, true);
+
+    // replaceLabelsOnNode
+    String[] replaceLabelsOnNodeArgs = { "-replaceLabelsOnNode" ,
```

Comment:
./hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/test/java/org/apache/hadoop/yarn/client/cli/TestRMAdminCLI.java:1108:    String[] replaceLabelsOnNodeArgs = { "-replaceLabelsOnNode" ,:40: '{' is followed by whitespace. [NoWhitespaceAfter]
./hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/test/java/org/apache/hadoop/yarn/client/cli/TestRMAdminCLI.java:1108:    String[] replaceLabelsOnNodeArgs = { "-replaceLabelsOnNode" ,:65: ',' is preceded with whitespace. [NoWhitespaceBefore]
./hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/test/java/org/apache/hadoop/yarn/client/cli/TestRMAdminCLI.java:1112:    String[] refreshQueuesArgs = { "-refreshQueues" , "-subClusterId", "SC-1"  };:34: '{' is followed by whitespace. [NoWhitespaceAfter]
./hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/test/java/org/apache/hadoop/yarn/client/cli/TestRMAdminCLI.java:1112:    String[] refreshQueuesArgs = { "-refreshQueues" , "-subClusterId", "SC-1"  };:53: ',' is preceded with whitespace. [NoWhitespaceBefore]
./hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/test/java/org/apache/hadoop/yarn/client/cli/TestRMAdminCLI.java:1115:    String[] refreshNodesResourcesArgs = { "-refreshNodesResources" , "-subClusterId", "SC-1"  };:42: '{' is followed by whitespace. [NoWhitespaceAfter]
./hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/test/java/org/apache/hadoop/yarn/client/cli/TestRMAdminCLI.java:1115:    String[] refreshNodesResourcesArgs = { "-refreshNodesResources" , "-subClusterId", "SC-1"  };:69: ',' is preceded with whitespace. [NoWhitespaceBefore]

Reply:
Thanks for your suggestion! I will fix it.